index.html

```diff
@@ -38,6 +34,149 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
             </ul>
         </section>
     </header>
+    <!-- cord section start from here -->
+    <section class="card-section">
+        <div class="card">
+            <div class="card-heading">
+                <img src="images/card.png" alt="card">
+            </div>
+            <div class="card-body">
+                <h1 class="card-title">Tonic</h1>
+                <div class="card-author">
+                    <h5 class="heading5">CANOPY</h5>
+                    <div class="dot"></div>
+                    <p>Back End Dev</p>
+                    <div class="dot"></div>
+                    <p>2015</p>
+                </div>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+            </div>
+            <div class="card-languages">
+                <p class="language-item ">html</p>
+                <p class="language-item ">css</p>
+                <p class="language-item ">javascript</p>
```

Comment:
- [ ] Good job so far but Please use a  `ul` and each of them a `li` for the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") for semantics reasons :+1: